src/lib/containers/entity_finder/tree/TreeNode.tsx

```diff
@@ -11,14 +11,13 @@ import { EntityType } from '../../../utils/synapseTypes/EntityType'
 import { EntityBadgeIcons } from '../../EntityBadgeIcons'
 import { EntityTypeIcon } from '../../EntityIcon'
 import { Map } from 'immutable'
-import Typography from '../../../utils/typography/Typography'
 
 export type RootNodeConfiguration = {
   nodeText: string
   children: (Pick<EntityHeader, 'name' | 'id' | 'type'> | ProjectHeader)[]
 }
 
-export enum NodeAppearance {
+export enum EntityTreeNodeType {
```

Comment:
Gave this a more appropriate name because there's a difference in behavior, and the previous enumeration name implied there was just a difference in appearance.